src/components/Layout/Header/SideNavContent.tsx

```diff
@@ -1,21 +1,10 @@
-import { ChatIcon, SunIcon } from '@chakra-ui/icons'
-import {
-  Box,
-  Flex,
-  FlexProps,
-  HStack,
-  Link,
-  Stack,
-  useColorMode,
-  useColorModeValue,
-  useMediaQuery
-} from '@chakra-ui/react'
-import { FaMoon } from 'react-icons/fa'
+import { ChatIcon } from '@chakra-ui/icons'
+import { Box, Flex, FlexProps, Link, Stack, useMediaQuery } from '@chakra-ui/react'
+import { IoIosSettings } from 'react-icons/io'
```

Comment:
is there a similar icon in font awesome we can use? the offset is just every so slightly different and it aligns weird

Reply:
Yeah!